examples/wakaama/dtlsconnection.c

```diff
@@ -448,7 +448,11 @@ dtls_connection_t *connection_create(dtls_connection_t *connList,
    char *host;
    char *tmp;
    char *port;
-   sock_udp_ep_t remote = { .family = AF_INET6 };
+   char *defaultport;
+   sock_udp_ep_t remote = {
+      .family = AF_INET6,
```

Comment:
Minor comment, you can use 
`sock_udp_ep_t remote = SOCK_IPV6_EP_ANY`

See [here](http://riot-os.org/api/sock_8h_source.html)